### PR TITLE
Fix "Deprecated selector in `fizzy/index.less`" errors

### PR DESCRIPTION
**Pull request by pauloelias**
@jglovier this PR should fix the deprecation errors folks are seeing

### CODE REVIEW

**Comment by jglovier** [comment]
Thank you so much, @pauloelias!!!! :heart: You are an absolute saint. :zap:

**Comment by pauloelias** [comment]
@jglovier glad I could help!

**Comment by andrew** [comment]
Can you tag the latest release so atom picks up the update?

**Comment by jglovier** [comment]
Hmm, so this appears to have broken the theme:

image

Also, shame on me for not checking out the branch and inspecting it locally before merging. 😭

**Comment by jglovier** [comment]
It appears to just be missing a declaration to set the background color of the `atom-text-editor` element to white.

File: styles/base.less [hunk 1]
```diff
@@ -5,7 +5,7 @@
 @import "ui-variables";
 @import "colors";
 
-.atom-text-editor, :host {
+.atom-text-editor, .atom-text-editor {
```
Comment (jglovier) [inline styles/base.less]: Any reason this selector is listed twice on this line?

**Review by jglovier — COMMENTED** [review]
Looks like we're still setting the background color correctly on the appropriate element, so I'm not sure why it's not being recognized.

**Comment by jglovier** [comment]
Hah. I figured it out. We're using the old class selector, but it's not named by that class anymore, it's a custom element.

File: styles/base.less (hunk 1 above)
Comment (pauloelias) [inline styles/base.less]: @jglovier I did this bases on my google-fu in trying to get things updated... This is my first time editing an Atom theme so I have no idea what I am doing :P 

Although during my dev/testing it seemed to work okay.

Comment (jglovier) [inline styles/base.less]: Okay no worries! I got it all cleaned up. Thanks again for doing all the heavy lifting on this one!

**Comment by jaredreich** [comment]
This theme has broken Atom's fonts - reproducable by trying to set fonts anywhere in Atom, styles.less, config.json, or with the [fonts package](https://atom.io/packages/fonts). Nothing works while this version of Fizzy is enabled.

**Pull request merged**